crates/core/src/kernel/snapshot/stream.rs

```diff
@@ -89,6 +89,7 @@ impl<O: Send + 'static> ReceiverStreamBuilder<O> {
 
     /// Spawn task that will be aborted if this builder (or the stream
     /// built from it) are dropped
+    #[allow(dead_code)]
```

Comment:
Crate: crates/core File: crates/core/src/kernel/snapshot/stream.rs (ReceiverStreamBuilder) Clippy warning fixed: "method spawn is never used --> crates/core/src/kernel/snapshot/stream.rs:92:12" Change: added #[allow(dead_code)] to the unused spawn method to silence the dead-code lint.

Reply:
Not changed since I don't want to potentially break the API.